CONTRIBUTING.md

```diff
@@ -113,6 +113,21 @@ following rules before you submit a pull request:
    scale in dimensionality: n_features is expected to be lower than
    100".
 
+A Note about CI:
```

Comment:
I'm not sure this is the right place. It doesn't fit before "You can also check for common errors".

Please change the heading to "Continuous Integration (CI)" and introduce it with a sentence saying "We use a few tools to ensure the integrity of our code and documentation for each commit in pull requests and merged into the master branch."

I also think this is more detail than necessary in Contributing.md and would happily see it only in the HTML guide.